acceptancetests/assess_persistent_storage.py

```diff
@@ -223,6 +222,63 @@ def assess_charm_removal_single_block_and_filesystem_storage(client):
     else:
         log.info('Detach single block device storage output check: {}'.format(
             detach_single_blk_output_check))
+    # storage status change after remove-application takes some time.
+    # from experiments even 30 seconds is not enough.
+    time.sleep(60)
+    # check the real status of storage after remove-application
+    storage_output = json.loads(
+        client.get_juju_output('storage', '--format', 'json', include_e=False))
+    storage_list = storage_output['storage'].keys()
+    if len(storage_list) != 1:
+        log.error('\n'.join(storage_list))
+        raise JujuAssertionError(
+            'Unexpected number of storage unit(s). '\
+            'Found: {}\nExpected: 1\ntest terminated.'.format(
```

Comment:
No need to say test terminated after raising an error. This is because we've only raised the error, and it could be caught and dealt with differently than we assume.